### DISCUSSION TITLE
ASUS ARMORY CRATE - 3.5

### DISCUSSION THREAD

**Discussion by cesarhfborges** [mathoudebine/turing-smart-screen-python#604] [Themes]
Sharing my Custom Asus Armory Crate theme.
evolution and construction

![Captura de tela 2024-11-12 111228](https://github.com/user-attachments/assets/073b60ac-0793-4554-b06f-220bec3f9f07)
[asus-armory-crate.zip](https://github.com/user-attachments/files/17717804/asus-armory-crate.zip)

**Comment by DJRoby19** [comment]
Very nice, shame it's not for Turing smart screen

**Reply by rep96-sudo** [reply]
Why is the GPU frequency not working?

**Reply by c0ldJS** [reply]
The background PNG could be lifted and made into a theme that will work with this project. But sharing the theme here with a file for a stock panel driver is confusing.
